src/auto-trace.js

```diff
@@ -100,7 +100,9 @@ export function withoutStacktrace(rawError, ...extraMiddlewares){
  */
 function wrapObjectWithError(err, stacktraceErr) {
```

Comment:
Maybe just pull the private function out into a helper file. Then test the helper?